test/specs/validate-address.spec.js

```diff
@@ -575,7 +575,7 @@ describe("validateAddress()", () => {
       country: "US",
       street: ["4 JERSEY ST"],
       cityLocality: "METROPOLIS",
-      stateProvince: "ME",
+      stateProvince: "TX",
```

Comment:
In the old simengine implementation, the state would default to the first two characters of the city specified. I didn't implement this in the new version and instead opted for a default but valid state.

Reply:
When you say that you default to a "valid state", do you take the country into account?  I'm fine with defaulting to "TX" when the country is "US", but then we should have other defaults for other countries.

Reply:
No, we do not but we could? Why do we do this BTW? Do we even want to support this type of defaulting for the Simengine?

Reply:
We're trying (as best we can) to match the behavior of the real ShipEngine API.  In the real API, users can pass-in an address without a `stateProvince` value, and if the address is valid then the API will automatically populate the `stateProvince` for them.  The same is also true of the `cityLocality` and `postalCode` fields.  

So we need to simulate this behavior in SimEngine.  Ideally, we'd try to be as realistic as possible by populating these fields with values that are valid for the given country.  This means we'd need an internal map of countries and their default city, state, and postal codes.  Totally doable, albeit a bit tedious.

Reply:
Roger that. I'm going to open a new ticket to do a more realistic implementation of this and we can work it into the schedule.